MessengerOnFileForLinux/ChatStarter/src/ChatRequest.cpp

```diff
@@ -93,10 +93,13 @@ std::unique_ptr<std::string> ChatRequest::getUserStatus(const std::string& usern
 bool ChatRequest::isUserActive(const User& user) const
 {
     log_.function("ChatRequest::isUserActive() started");
+    const std::string command = "ps -u " + user.getUsername() + " | grep messenger_bin";
+    const std::string commandOutput = ConsolControl::getStdoutFromCommand(command);
     std::unique_ptr<std::string> userStatusToCompare = getUserStatus(user.getUsername());
 
-    if (nullptr == userStatusToCompare)
+    if (commandOutput.empty() || nullptr == userStatusToCompare)
```

Comment:
rozbić na dwa ify, bo teraz zespusuje dzialanie programu

Reply:
Jeżeli tak to widzisz to zadam inne pytanie - czy to jest miejsce, w którym powinniśmy obsługiwać sprzątanie po użytkowniku?

Reply:
Jedyne w którym możemy posprzątać po użytkowniku który nie prowadził rozmowy, a został wyrzucony z serwera przez nieobsługiwany sygnał.

Reply:
Uważam, że to sprzątanie powinno być wyjęte do inne klasy i być po cichu gdzieś uruchamiane.

Reply:
Ok, tutajbym już bym tego nie ruszał, a zrobiłbym na to taska na 1SP na ten sprint na mnie. Co ty na to ?

Reply:
Albo nie, zrobie to już tutaj i wystawie nowe review